examples/client/Locomotion/src/pages/ActiveRide/index.js

```diff
@@ -321,12 +331,22 @@ const RidePage = ({ mapSettings, navigation }) => {
     };
   }, []);
 
+  const loadRides = async () => {
+    const d = new Date();
+    d.setMonth(d.getMonth() - 1);
+    const rides = await fetchRides({ fromDate: d });
```

Comment:
Please use the same logic as here to look for outstanding balance ride, this is not good enough...